server/routes/referrals/referralStartForm.test.ts

```diff
@@ -0,0 +1,32 @@
+import { Request } from 'express'
+import ReferralStartForm from './referralStartForm'
+
+describe(ReferralStartForm, () => {
+  describe('isValid', () => {
```

Comment:
Can we test `error` too? I know it's a pain because it's basically a duplicate of this test. But I don't know of a better way of testing this kind of situation, where you have two public methods, one of which is directly derived from the other with minimal additional logic. So far we've tested both methods in our form tests. Do you have any thoughts on this?

Reply:
Yeah, that's why I didn't bother creating a test for it, but I think the test for `error` is probably more useful than the test for `isValid` as it returns an object and is the dependency for `isValid`. It's a bit tricky, I think it's probably fine to just duplicate the tests.